keyboards/le_chiffre/keymaps/ghidalgo93/rules.mk

```diff
@@ -0,0 +1,2 @@
+OLED_DRIVER_ENABLE = no
+EXTRAKEY_ENABLE = yes
```

Comment:
- `OLED_DRIVER_ENABLE` has been deprecated for a long time now.
- `EXTRAKEY_ENABLE` is enabled at the keyboard level.
So, this file should just be removed.